db/c_test.c

```diff
@@ -1461,6 +1461,18 @@ int main(int argc, char** argv) {
     rocksdb_cuckoo_options_destroy(cuckoo_options);
   }
 
+  StartPhase("options");
+  {
+    rocksdb_options_t* o;
+    o = rocksdb_options_create();
+
```

Comment:
Please change at least one setting in `o` before copying, and assert the change is reflected in `copy`.

Reply:
It was my initial intention, but as far as I can see, all comparison checks use `CheckEqual` that operates on strings (`char*`). Additionally, there are no getters for options in the C API.

Reply:
Oh, right. You can do the naughty thing based on create_if_missing being the first field or at least near the beginning. Set create_if_missing to true, make a copy, CheckCondition+memcmp that the first ~16 bytes at each pointer (after cast) are same. And then set create_if_missing on one to false, and make sure CheckCondition+memcmp shows a difference.

Although there's a possibility of a future "false positive" test failure, I think it's manageable and preferable to leaving open false negative.

Reply:
Done.